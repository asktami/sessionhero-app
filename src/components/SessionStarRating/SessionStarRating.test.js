import React from 'react';
import ReactDOM from 'react-dom';
import { SessionStarRating } from './SessionStarRating';

it('renders without crashing', () => {
	const div = document.createElement('div');
	const props = {
		rating: 3
	};
	ReactDOM.render(<SessionStarRating {...props} />, div);
	ReactDOM.unmountComponentAtNode(div);
});
