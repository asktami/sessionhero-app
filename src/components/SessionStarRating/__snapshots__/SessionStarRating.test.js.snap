// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SessionStarRating component renders SessionStarRating by default 1`] = `
<span
  className="SessionStarRating"
>
  <Star
    filled={true}
    key="0"
  />
  <Star
    filled={true}
    key="1"
  />
  <Star
    filled={true}
    key="2"
  />
  <Star
    filled={false}
    key="3"
  />
  <Star
    filled={false}
    key="4"
  />
</span>
`;

exports[`SessionStarRating component renders the SessionStarRating given props 1`] = `
<span
  className="SessionStarRating"
>
  <Star
    filled={true}
    key="0"
  />
  <Star
    filled={true}
    key="1"
  />
  <Star
    filled={true}
    key="2"
  />
  <Star
    filled={false}
    key="3"
  />
  <Star
    filled={false}
    key="4"
  />
</span>
`;
